src/Row.js

```diff
@@ -46,7 +46,7 @@ var Row = React.createClass({
   render(): ?ReactElement {
     var className = joinClasses(
       'react-grid-Row',
-      "react-grid-Row--${this.props.idx % 2 === 0 ? 'even' : 'odd'}"
+      'react-grid-Row--' + this.props.idx % 2 === 0 ? 'even' : 'odd'
```

Comment:
nice, yes we'll be releasing a style guide for contributing soon, and we'll enforce that with a linting build task. For the moment I will merge this in and will review all string interpolation shortly